src/cozy.js

```diff
@@ -63,6 +63,7 @@ module.exports = (() => ({
             updatedAt: {
               $gt: lastSync
             }
+            //TODO: filter by contact account id / email -> only the second time, otherwise no contacts come up
```

Comment:
Why "only the second time" ?

Reply:
The first time the connector runs, we don't have a `lastSync` and we want to get all contacts. This note is more a reminder, we're not quite clear about what the implications of this are.